pkg/scheduler/plugins/nodegroup/nodegroup.go

```diff
@@ -0,0 +1,229 @@
+/*
+Copyright 2019 The Kubernetes Authors.
```

Comment:
please update the copyright to be the right one.

Reply:
fixed